docker-compose.yml

```diff
@@ -7,7 +7,6 @@ services:
       DEBUG:
       BUILDKITE:
       BUILDKITE_PIPELINE_NAME:
-      UNITY_VERSION:
```

Comment:
This still seems to be used elsewhere in the pipeline yaml: https://github.com/bugsnag/bugsnag-unity/pull/423/files#diff-a4cd708cc6050e0ba4f2289e6d602a54f79cba40363cbf890d8ff9ceade4c373R564

Is it definitely not required anymore?

Reply:
Good spot - the majority of uses in pipeline steps are in a non-Docker context (Mac and Windows build servers).  There was one instance where it was being passed to a Maze Runner container, but it was never required and presumably just a copy-paste error from early on (so I have removed it).